### DISCUSSION TITLE
Investigate support for radiometric and photometric quantities conversion.

### DISCUSSION THREAD

**Discussion by thomasmansencal** [colour-science/colour#678] [General]
Every so often I need to perform conversion between radiometric and photometric quantities, I would like to investigate support for that. @MichaelMauderer is it something that makes sense to you?

**Comment by MichaelMauderer** [comment]
Makes sense to me. Since we are already dealing with pre-perception physical quantities (e.g.EM spectra) this doesn't seem out of scope.

**Comment by thomasmansencal** [comment]
Excellent, as part of the spectral refactor we could add units to those, even if just as an indication.

There are maybe existing packages that handle that, I'll do a survey and see if it can be hooked without pain.

**Comment by thomasmansencal** [comment]
Noticeable packages I remember are:
- [Pint](https://pint.readthedocs.org/en/0.7.2/)
- [Quantities](https://pythonhosted.org/quantities/user/tutorial.html)

Pint seems to be more mature.

**Comment by thomasmansencal** [comment]
I was recently toying with the following spreadsheet that covers the typical stuff I need in my field: https://docs.google.com/spreadsheets/d/19cl0XvJIArSIku-XxoDyELH1dYIcK9R7sWQxuEgJGh8/edit?usp=sharing
- Photometry (and Radiometry)
- Photography
- Quantization
- Resolution (Visual acuity)

Some of that stuff could be part of the API at some point.
